realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -531,6 +531,7 @@ private static String getManagementRealmUrl(URL authUrl) {
      * @return
      */
     public PermissionManager getPermissionManager() {
+        new AndroidCapabilities().checkCanDeliverNotification("The PermissionManager can only opened on a Looper thread.");
```

Comment:
Commit log and PR description does not have a mention about this change.
Is it an intentional change?